migration-toolkit-for-containers/upgrading-mtc.adoc

```diff
@@ -0,0 +1,19 @@
+[id="upgrading-3-4"]
+= Upgrading the Migration Toolkit for Containers
+include::modules/common-attributes.adoc[]
+:context: upgrading-3-4
+
+toc::[]
+
+You can upgrade the {mtc-full} ({mtc-short}) by using the {product-title} web console.
+
+[IMPORTANT]
+====
+You must ensure that you upgrade to the same {mtc-short} version on all clusters.
+
+If you are upgrading {mtc-short} version 1.3, you must perform an additional procedure to update the `MigPlan` custom resource (CR).
```

Comment:
"If you are upgrading MTC version 1.3" - upgrading to 1.3, or from 1.3?

Reply:
I think it can only mean "upgrading from". You upgrade "old shoes" to "new shoes".